media/js/mozorg/manifesto.js

```diff
@@ -32,7 +32,7 @@ $(function() {
         var href = $(this).attr('href');
 
         // Open Twitter in a sub window
-        event.preventDefault();
         openTwitterSubwin(href);
+        event.preventDefault();
```

Comment:
One other small tweak that may help with Safari on desktop opening two windows, is changing the event listener above from `document` to the buttons themselves: 

```javascript
$('.js-manifesto-share').on('click', function (event) {
    // foo
});
```